__tests__/main.test.js

```diff
@@ -135,4 +145,48 @@ describe('Articles', () => {
 			.send({ inc_votes: 'not-number' })
 			.expect(400)
 	})
+	it('POST 201 /api/articles/article_id/comments responds with added comment when posted', () => {
+		const body = {
+			body: 'We can do comments while testing do not we?',
+			author: 'icellusedkars',
+		}
+		return request(app)
+			.post('/api/articles/1/comments')
+			.set('Accept', 'application/json')
+			.send(body)
+			.expect(201)
+			.then(({ body: { comment } }) => {
+				expect(comment.body).toBe(body.body)
+				expect(comment.author).toBe(body.author)
+				expect(comment.article_id).toBe(1)
+			})
+	})
+	it('POST 400 /api/articles/article_id/comments responds with bad request when poseted non-exist author name', () => {
+		const body = {
+			body: 'We can do comments while testing do not we?',
+			author: 'iAmNotinUsers',
+		}
+		return request(app)
+			.post('/api/articles/1/comments')
+			.set('Accept', 'application/json')
+			.send(body)
+			.expect(400)
+			.then(({ body: { msg } }) => {
+				expect(msg).toBe('Bad request')
+			})
+	})
+	it('POST 400 /api/articles/article_id/comments responds with bad request when body is empty or non-string type or invalid id', () => {
```

Comment:
This can't be all one test, and you aren't testing these behaviours. You're only testing if the body is not a string.

You still need to test:
- posting to a non-existent id
- posting to an invalid id
- possibly missing fields as well

In addition to these ones that you've listed but haven't implemented, I'd like to see a test for sending a request with an unnecessary property (unnecessary properties should be ignored)